.travis.yml

```diff
@@ -8,12 +8,11 @@ script:
     -e MYSQL_ROOT_PASSWORD=123456
     --tmpfs=/var/lib/mysql/:rw,noexec,nosuid,size=600m
     --tmpfs=/tmp/:rw,noexec,nosuid,size=50m mysql:5.6 &&
-    
+
     docker run
     --link database:database
     -v $(pwd):/repo
     -e CI=true
-    -e PROJECT=hollowverse-api
```

Comment:
This one is needed here (and all other Elastic Beanstalk repos). It's used in `deploy.js` to determine the application name to deploy to.